src/integrationTest/java/uk/gov/ons/fwmt/fwmtrmadapter/IntegrationTestConfig.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.ons.fwmt.fwmtrmadapter;
+
+import org.springframework.amqp.rabbit.connection.ConnectionFactory;
+import org.springframework.amqp.rabbit.listener.SimpleMessageListenerContainer;
+import org.springframework.amqp.rabbit.listener.adapter.MessageListenerAdapter;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.ons.fwmt.fwmtrmadapter.helper.TestReceiver;
+
+@Configuration
+public class IntegrationTestConfig {
+
+  static final String adapterToJobSvcQueue = "adapter-jobSvc";
```

Comment:
Move all the queue and exchange name to one locations and access it across the applications.
Also if possible externalise it.